common/decklist.cpp

```diff
@@ -726,12 +726,29 @@ bool DeckList::deleteNode(AbstractDecklistNode *node, InnerDecklistNode *rootNod
 void DeckList::updateDeckHash()
 {
     QStringList cardList;
-    for (int i = 0; i < root->size(); i++) {
+    bool isValidDeckList = true;
+    QSet<QString> hashZones, optionalZones;
```

Comment:
Too much of a pain to get it static in an outside class, and due to its nature I think it's fine how it is